docs/tutorials/google/floquet_calibration_example.ipynb

```diff
@@ -39,7 +39,7 @@
     "id": "AF-CFuvzPx4x"
```

Comment:
<p>I know this isn't changed in the current PR, but: </p><p> </p><p>this circuit diagram wraps really badly on my machine because of the use of <code>print</code>  because you want to set the qubit order. If you just have the circuit as the last thing in the cell, jupyter will print its text representation in a cell with a horizontal scroll bar. I don't know if there's any fix here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Thanks for pointing out, I also don't know of a fix. At least it appears nicely on the website with a horizontal scroll bar.</p><p> </p><p>Worth noting: Not specifying a qubit order makes the circuit much harder to understand here, even if it displays nicely.</p>